doc/src/manual/mathematical-operations.md

```diff
@@ -366,6 +366,7 @@ Julia applies the following order and associativity of operations, from highest
 | Syntax         | `<\|`                                                                                             | Right                      |
 | Comparisons    | `> < >= <= == === != !== <:`                                                                      | Non-associative            |
 | Control flow   | `&&` followed by `\|\|` followed by `?`                                                           | Right                      |
+| Syntax         | `=>`                                                                                              | Left                       |
```

Comment:
This could say `Pair`, and I believe it is right associative.

Reply:
I updated the PR according to your remarks; yes `1 => 2 => 3` gives `1 => (2=>3)`.